backend/LexBoxApi/GraphQL/CustomTypes/OrgGqlConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿using LexCore.Entities;
+
+namespace LexBoxApi.GraphQL.CustomTypes;
+
+[ObjectType]
+public class OrgGqlConfiguration : ObjectType<Organization>
+{
+    protected override void Configure(IObjectTypeDescriptor<Organization> descriptor)
+    {
+        descriptor.Field(p => p.CreatedDate).IsProjected();
```

Comment:
Does anything else need to be here? Does this file even need to exist, or can it be safely deleted?

Reply:
this file is not required, however since you added it you might as well keep it around as we might need it for future tweaks